cyences_app_for_splunk/default/savedsearches.conf

```diff
@@ -3124,12 +3153,69 @@ search = | tstats `cs_summariesonly_network_traffic` sum(All_Traffic.bytes) as t
+| `drop_dm_object_name(All_Traffic)` \
+| eval total_MB = round(total_bytes/(1024*1024),2) | fields - total_bytes \
+| eval total_m_packets = round(total_packets/1000000,2) | fields - total_packets \
+| eval original_event=1 \
+| appendpipe \
+    [| stats sum(total_m_packets) as total_m_packets_ by sourcetype, src_ip \
+    | sort - total_m_packets_ \
+    | eval src_ip_mix=src_ip." (".total_m_packets_." M packets)" \
+    | fields - total_m_packets_] \
+| appendpipe \
+    [| where original_event==1 \
+    | stats sum(total_m_packets) as total_m_packets_ by sourcetype, action \
+    | sort - total_m_packets_ \
+    | eval action_mix=action." (".total_m_packets_." M packets)" \
+    | fields - total_m_packets_] \
+| stats sum(total_m_packets) as total_m_packets, sum(total_MB) as total_MB, list(src_ip_mix) as src_ip_mix, list(action_mix) as actions by sourcetype \
+| eval top5_src_ip=mvindex(src_ip_mix,0,4) | fields - src_ip_mix \
+| where total_m_packets>`cs_network_outbound_min_m_packets` \
+| lookup cs_outbound_network_traffic_upperbound.csv sourcetype \
+| where total_m_packets>upperBound_total_m_packets OR total_MB > upperBound_total_MB \
+| eval cyences_severity = case(total_m_packets>upperBound_total_m_packets*3 OR total_MB>upperBound_total_MB*3, "critical", total_m_packets>upperBound_total_m_packets*2 OR total_MB>upperBound_total_MB*2, "high", true(), "medium") \
+| table sourcetype, cyences_severity, total_m_packets, avg_total_m_packets, upperBound_total_m_packets, actions, top5_src_ip, total_MB, avg_total_MB, upperBound_total_MB \
+| fieldformat avg_total_m_packets=avg_total_m_packets." M" \
+| fieldformat upperBound_total_m_packets=upperBound_total_m_packets." M" \
+| fieldformat total_m_packets=total_m_packets." M" \
+| `cs_network_unusual_outbound_traffic_filter`
+action.cyences_notable_event_action = 1
+action.cyences_notable_event_action.param.filter_macro_name = cs_network_unusual_outbound_traffic_filter
+action.cyences_notable_event_action.contributing_events = | datamodel Network_Traffic All_Traffic search strict_fields=false | `drop_dm_object_name(All_Traffic)` | search `cs_private_ips(src_ip)` AND `cs_public_ips(dest_ip)`
+action.cyences_notable_event_action.system_compromised_search = | stats values(total_m_packets) as total_m_packets, values(total_MB) as total_MB by sourcetype
+action.cyences_notable_event_action.system_compromised_drilldown = | datamodel Network_Traffic All_Traffic search strict_fields=false | search sourcetype=$row.sourcetype$ | `drop_dm_object_name(All_Traffic)` | search `cs_private_ips(src_ip)` AND `cs_public_ips(dest_ip)`
+action.cyences_send_email_action = 1
+action.cyences_notable_event_action.products = Cisco IOS, FortiGate, Palo Alto, Sophos Firewall, Windows DNS
```

Comment:
Windows DNS - I don't think this should be here.

Reply:
Yes, windows DNS mapped with Network Resolution not with network traffic.

I'll remove it.